test/rubygems/test_require.rb

```diff
@@ -106,8 +100,6 @@ def test_concurrent_require
     assert t1.join, "thread 1 should exit"
     assert t2.join, "thread 2 should exit"
   ensure
-    return if $! # skipping
-
     Object.send :remove_const, :FILE_ENTERED_LATCH
```

Comment:
should this now have an `if defined?`

Reply:
Thanks. It caused failure with Ruby 1.8 maybe. I added a condition.